pom.xml

```diff
@@ -15,14 +15,14 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <kotlin.compiler.jvmTarget>1.8</kotlin.compiler.jvmTarget>
-        <smack.version>4.2.4-47d17fc</smack.version>
+        <smack.version>4.4.3</smack.version>
         <jersey.version>2.30.1</jersey.version>
         <jackson.version>2.9.5</jackson.version>
         <junit.version>5.2.0</junit.version>
         <kotest.version>4.4.3</kotest.version>
         <mockk.version>1.10.0</mockk.version>
         <ktor.version>1.5.2</ktor.version>
-        <jicoco.version>1.1-54-g5387346</jicoco.version>
+        <jicoco.version>1.1-94-g9c140b8</jicoco.version>
```

Comment:
This is a little scary just because of the amount of changes. Do we trust the tests we have to catch any potential issues?

Reply:
I don't know - I see the Jenkins tests ran, but I also see that the jibri Jenkins tests only take 10 seconds so I can't say I'm terribly confident in them.